client/galaxy/scripts/nls/fr/locale.js

```diff
@@ -0,0 +1,272 @@
+
+// // ---- history-view-annotated
+"Dataset" :
+    "Jeu de données",
+"Annotation" :
+    "Annotation",
+
+// ---- history-view-edit-current
+"This history is empty. Click 'Get Data' on the left tool menu to start" :
+    "Cet historique est vide. Cliquer sur 'Téléverser des données' au niveau du menu d'outils à gauche pour démarrer",
+"You must be logged in to create histories" :
+    "Vous devez être connecté pour créer un historique",
+//TODO:
+//"You can <% loadYourOwn %> or <% externalSource %>" :
+//"Vous pouvez <% loadYourOwn %> ou <% externalSource %>",
+"load your own data" :
+    "Charger vos propres données",
+"get data from an external source" :
+    "Charger des données depuis une source externe",
+
+// these aren't in zh/ginga.po and the template doesn't localize
+"Include Deleted Datasets" :
+    "Inclure les jeux de données supprimés",
+"Include Hidden Datasets" :
+    "Inclure les jeux de données cachés",
+
+
+// ---------------------------------------------------------------------------- datasets
+// ---- hda-model
+"Unable to purge dataset" :
+    "Impossible de purger le jeu de données",
+
+// ---- hda-base
+// display button
+"Cannot display datasets removed from disk" :
+    "Impossible de visualiser les jeux de données supprimés du disque",
+"This dataset must finish uploading before it can be viewed" :
+    "Le jeu de données doit être totalement téléversé avant de pouvoir être visualiser",
+"This dataset is not yet viewable" :
+    "Ce jeu de données n'est pas visualisable",
+"View data" :
+    "Voir les données",
+// download button
+"Download" :
+    "Télécharger",
+"Download dataset" :
+    "Télécharger le jeu de données",
+"Additional files" :
+    "Fichiers additionnels",
+// info/show_params
+"View details" :
+    "Voir les détails",
+
+// dataset states
+// state: new
+"This is a new dataset and not all of its data are available yet" :
+    "Il s'agit d'un nouveau jeu de données et seule une partie des données est accessible pour le moment",
+// state: noPermission
+"You do not have permission to view this dataset" :
+    "Vous n'avez pas la permission de voir ce jeu de données",
+// state: discarded
+"The job creating this dataset was cancelled before completion" :
+    "Le calcul ayant créé ce jeu de données a été annulé prématurément",
+// state: queued
+"This job is waiting to run" :
+    "Ce calcul est en attente de traitement",
+// state: upload
+"This dataset is currently uploading" :
+    "Ce jeu de données est en cours de téléversement",
+// state: setting_metadata
+"Metadata is being auto-detected" :
+    "Les métadonnées sont auto-détectées",
+// state: running
+"This job is currently running" :
+    "Le traitement est en cours",
+// state: paused
+"This job is paused. Use the \"Resume Paused Jobs\" in the history menu to resume" :
+    "Ce traitement est en pause. Utilisez le \"Relancer les traitements en pause\" dans le menu d'historique pour le relancer",
```

Comment:
@lecorguille @yvanlebras I added more translation points in 54af1979676c817b00044e645383e638b86fdcfa though unfortunately not for "Get Data" yet -- that one's more complicated and I'll want to address it in a future PR.   If you guys want to fill in those, we can swap "Resume Paused Jobs", etc.